### PR TITLE
#patch Feature/demo module

### PR DESCRIPTION
**Pull request by Greatmar2**
Finish TodoList and TodoItem endpoints

### CODE REVIEW

File: EngineBay.DemoModule/Queries/QueryTodoItem.cs [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    using System.Threading.Tasks;
+    using EngineBay.Core;
+    using LinqKit;
+    using Microsoft.EntityFrameworkCore;
+
+    public class QueryTodoItem : PaginatedQuery<TodoItem>, IQueryHandler<QueryTodoItemRequest, PaginatedDto<TodoItemDto>>
+    {
+        private readonly DemoModuleDbContext dbContext;
+
+        public QueryTodoItem(DemoModuleDbContext dbContext)
```
Comment (windy-worm) [inline EngineBay.DemoModule/Queries/QueryTodoItem.cs]: DemoModuleQueryDbContext?

File: EngineBay.DemoModule/Queries/QueryTodoItem.cs [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+{
+    using System.Globalization;
+    using System.Linq.Expressions;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using EngineBay.Core;
+    using LinqKit;
+    using Microsoft.EntityFrameworkCore;
+
+    public class QueryTodoItem : PaginatedQuery<TodoItem>, IQueryHandler<QueryTodoItemRequest, PaginatedDto<TodoItemDto>>
```
Comment (windy-worm) [inline EngineBay.DemoModule/Queries/QueryTodoItem.cs]: Why don't we pass PaginationParameters directly? Do we anticipate this request growiung?

**Review by windy-worm — APPROVED** [review]

File: EngineBay.DemoModule/Queries/QueryTodoItem.cs (hunk 2 above)
Comment (Greatmar2) [reply EngineBay.DemoModule/Queries/QueryTodoItem.cs]: Perhaps the filtering, though I'm looking into that now

File: EngineBay.DemoModule/Queries/QueryTodoItem.cs (hunk 1 above)
Comment (Greatmar2) [reply EngineBay.DemoModule/Queries/QueryTodoItem.cs]: Good catch

**Pull request merged**